src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -16,6 +16,22 @@
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
 const createAuthenticatorOptions = (options = []) => {
+  // Split OV into individual entries (one for each method).
+  const ovItem = options.find((option) => option.relatesTo.type && option.relatesTo.type === 'app');
```

Comment:
The idea makes sense to me.

1. You probably only want to change `getAuthenticatorsVerifyUiSchema`

2. Instead of using `JSON.parse(JSON.stringify`, I think we can be more explicit since we know the structure.

```
newOption = Object.assign([], option, {
  options: [item]
});
```

3. You had collapse `security_key` in `FormInputFactory`, does it make sense to move here as well? Just to be consistent

Reply:
For #1, I was thinking that the options array for methodType would only be available for verify, so we would check for that before altering the options.